### DISCUSSION TITLE
792

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#971] [Comments]
# 792

웹 소설 전지적 독자 시점을 몇 번이나 다시 읽을 정도로 좋아한다.이 소설의 세계관이 너무 마음에 들기 때문인데,그 인물이 어떤 서사를 쌓아왔는지에 따라 강함이 결정 된다.세계관 속 인물의 타고난 강함이 있더라도,계속해서 서사를 쌓지 않으면 그 인물은 계속해서 약해지기만 하며 시간이 더 지나면 소멸한다.더 많은 모험을 즐기고, 더 많은 사건을 거친 인물일수록 더 강해지고 오래오래 살 수 있다.그래서 12과업을 달성한 헤라클래스나 제천대성, 필마온등으로 수많은 이야기를 쌓은 손오공 등이 최강의 캐릭터로 묘사된다.이 세계관 속 주인공과 동료들은 어마어마한 험난한 여정을 걸어간다.더이상 스테이지를 클리어하지 않아도 평화롭게 지낼 수 있는 스테이지를 스스로 무너뜨려야 한다거나동료들에게 죽임을 당해야만 다음 스테..

https://jojoldu.tistory.com/792?category=689637

**Comment by Invidam** [comment]
<기억보다 기록을>의 <향로>처럼 ㅎㅎㅎ
